pom.xml

```diff
@@ -24,7 +24,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <prometheus.metrics.version>1.3.1</prometheus.metrics.version>
+        <prometheus.metrics.version>1.4.0-SNAPSHOT</prometheus.metrics.version>
```

Comment:
@dhoard thanks, for this change, I guess there is no other option, the client_java project uses snapshots.

Reply:
@gtully to clarify, by design, the `jmx_exporter` project only uses `client_java` release artifacts. There is no snapshot dependency between the projects.

It does become cumbersome when there are dependencies, but it only occurs rarely.

Reply:
thanks, that makes sense, this will have to wait for https://github.com/prometheus/client_java/issues/1088